content/handbook/security/corporate/end-user-services/laptop-management/laptop-security/updates/_index.md

```
@@ -0,0 +1,73 @@
+stale: false
+---
+
+## 概要
+
+私たちは、悪用可能な脆弱性のリスクを軽減するため、すべてのエンドポイントオペレーティングシステムとインストール済みソフトウェアでセキュリティパッチを最新の状態に保つことを信条としています。
+
+## 想定される事項
+
+2〜3 週間ごとに 30〜60 分間ソフトウェア更新を実行する計画を立ててください。これは、ノートパソコンを再起動する前に作業を保存しアプリケーションウィンドウを閉じることができるよう、勤務日の終わりや勤務週の終わりに向けて計画を立てるのが最も簡単です。
+
+新しい macOS バージョンがリリースされると、当社のセキュリティベータテスターが最新バージョンでテストを実行し、生産性に影響を与える破壊的変更がなくアップグレードがスムーズに進むことを確認します。1〜3 週間のテスト後（重大な脆弱性を除く）、[Jamf](/handbook/security/corporate/systems/jamf) と [Nudge](#nudge-for-macos-updates) を使用してデスクトップポップアップ通知を送信し、チームメンバーに次の数日以内の都合の良いタイミングでオペレーティングシステムを更新するよう促します。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)